go/vt/vtgate/vtgate.go

```diff
@@ -186,6 +186,8 @@ var (
 
 	vstreamSkewDelayCount = stats.NewCounter("VStreamEventsDelayedBySkewAlignment",
 		"Number of events that had to wait because the skew across shards was too high")
+
+	vindexUnknownParams = stats.NewGauge("VindexUnknownParams", "Number of params unrecognized by Vindexes")
```

Comment:
kindly change the metric name to `VindexUnknownParameters`

Reply:
Similar change would be needed in the release notes.

Reply:
Done! Updated website PR too